frontends/infinite-corridor/src/pages/HomePage.tsx

```diff
@@ -1,13 +1,12 @@
 import React, { useCallback, useState } from "react"
```

Comment:
Main changes in this file:
- using mui's responsive Container for layout
- changed a `div` to a `section`, and moved the h2 into the `section`.